kore/src/Kore/Step/Axiom/EvaluationStrategy.hs

```diff
@@ -239,10 +239,21 @@ applyFirstSimplifierThatWorks
               error
                   (  "Unexpected simplification result with more "
                   ++ "than one configuration: "
-                  ++ show applicationResult
+                  ++ (show . Pretty.vsep $
+                        [ "Simplification result with remainder:"
```

Comment:
Please change this to use the error message above, i.e. 
```suggestion
                        [ "Unexpected simplification result with more than one configuration:"
```

Reply:
done